cabal.project

```diff
@@ -60,3 +60,10 @@ package plutus-scripts-bench
 -- temporary! Please read the section in CONTRIBUTING about updating dependencies.
 
 -- `smtp-mail` should depend on `crypton-connection` rather than `connection`!
+
+source-repository-package
```

Comment:
This PR requires new cardano-ping release with: https://github.com/input-output-hk/ouroboros-network/pull/4743/files